### PR TITLE
Devanshukoli patch 1

### PR DESCRIPTION
**Pull request by Devanshukoli**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4216"></a>

### CODE REVIEW

File: data/Devanshukoli.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Devanshu Koli",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "A Recent Front-End Software Intern and Open-source contributor as well. Here to meet with nice people."
```
Comment (kumarsonsoff3) [inline data/Devanshukoli.json:5]: There is a missing comma `,` here. 
```suggestion
  "bio": "A Recent Front-End Software Intern and Open-source contributor as well. Here to meet with nice people.",
```

File: data/Devanshukoli.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+   {
```
Comment (kumarsonsoff3) [inline data/Devanshukoli.json:6]: Please add the missing links key here to validate your json. 

```suggestion
   "links": [ {
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Hey @Devanshukoli, Please make the below changes as suggested to validate your json + revert the changes from the `CHANGELOG.md` file. Let us know if you need any help with this. 

This video https://youtu.be/YUHS5Ana7Jw will help you with revert the changes.

**Comment by Devanshukoli** [comment]
@kumarsonsoff3 I have done as told. I guess. so please kindly view this if any further changes are required please say so!

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Still you need to revert the changes from the `CHANGELOG.md`. You can refer this video https://youtu.be/YUHS5Ana7Jw will help you with this. 

Feel free to ask if you need any help.

**Comment by Devanshukoli** [comment]
@kumarsonsoff3 hey, I actually tried to do as from the eddie's video. but Still am stuck. so could you help me?

**Comment by kumarsonsoff3** [comment]
> -So, i tried first, directly copying files from the main file(eddihub/linkfree) and that won't work? right?

@Devanshukoli, It also works because the one in your branch & base branch (EddieHubCommunity/linkfree) should be the same, then it will not be included in the PR. 

Don't worry, here you just need to add the last 2 lines as Panquesito7 suggested above. Should be good to go!

**Comment by Devanshukoli** [comment]
Hey @Panquesito7 
git
I am doing as you have told but in bash it said **fatal error**

**Comment by Panquesito7** [comment]
> Hey @Panquesito7 git I am doing as you have told but in bash it said **fatal error** -As earlier you told i just added missing lines, then in git bash commited and then tried to push origin test-doc but get above message...

You must clone and push to your fork instead, not the original repository.

**Review by kumarsonsoff3 — APPROVED** [review]
Fixed the CHANGELOG file for you. You can refer [these docs](https://linkfree.eddiehub.io/docs/how-to-guides/editing) if you need to make any other changes in your profile later. 🚀

**Comment by Devanshukoli** [comment]
Yeeey! My PR is finally merged. Thanks for guiding me this through. @kumarsonsoff3 @Panquesito7

**Pull request merged**